exam_2.py

```diff
@@ -0,0 +1,6 @@
+Dict={}
+a=input("Bir değer giriniz:")
+for i in range(1,int(a)):
+    print({i:i*i})

```

Comment:
Bu `i` degerlerini printlemek yerine olusturdugunuz sozluge eklemeniz gerekiyordu. my_dict isimli bir sozlugumuz oldugunu dusunursek (`my_dict = {}`), yazmamiz gereken kod:
```suggestion
     my_dict[i] = i*i
     

```